FINO_20100513to26/wrf_setup.sh

```diff
@@ -35,10 +35,10 @@ MAX_DOM="3"
 # DOMAIN CONFIG (default: "")
 DOM_STR=""
 # REANALYSIS TO USE (ERA, NARR, GFS, ERA5)
-REAN_STR="ERA5" 
+REAN_STR="GFS"
```

Comment:
So is this the preferred reanalysis to use now?

Reply:
Yeah - the initial check for "best performer" considered 18Z on the 16th to 00Z on the 17th. Since we are actually targeting the period of 20Z to 00Z, GFS was the "best performer" before. But then we found this issue with the grid sizes and had to change it all again. GFS is still the top dog for this period. Sensitivity tests are coming soon but I don't expect a big difference from those.